mongobin/getOrDownload.go

```diff
@@ -98,11 +101,24 @@ func GetOrDownloadMongod(urlStr string, cachePath string, logger *memongolog.Log
 			return "", fmt.Errorf("error reading from tar: %s", tarErr)
 		}
 
-		if strings.HasSuffix(nextFile.Name, "bin/mongod") {
-			break
+		if strings.HasSuffix(nextFile.Name, "bin/mongod") || strings.HasSuffix(nextFile.Name, "bin/mongo") {
+			fileCount++
+			s, err := saveFile(path.Join(dirPath, filepath.Base(nextFile.Name)), tarReader, logger)
+			if err != nil {
+				return s, err
+			}
+			if fileCount >= maxFilesCount {
+				break
+			}
 		}
 	}
 
+	logger.Infof("finished downloading mongod to %s in %s", mongodPath, time.Since(downloadStartTime).String())
+
+	return mongodPath, nil
```

Comment:
Clarify why mongodPath is got in this way rather than using saveFile

perhaps it is better for the saveFile function to just return error now? instead of (string, error)

Reply:
This PR is making the least amount of changes to fix the bugs pointed out in #13. I did not change any semantics. This means that this function returns exactly what it returned before this PR, which is the full path of `mongodPath`, that is computed by the line `mongodPath := path.Join(dirPath, "mongod")` above (43) and remains a constant regardless of execution path (as long as no error is returned). 

`saveFile` function is simply an extraction of the original code, which exclusively handled extracting `mongod` alone without also extracting the needed `mongo`. 

We could switch it to return just an error, and recompute `dirPath` at the call-site, but I think a better solution would be to return `dirPath` itself and add either `mongo` or `mongod` to the returned value at the call-site instead.

In any case, this is simply the minimum change required for this library to extract all required files, and intended as a workaround for my teams until a more proper PR is created and merged.